corehq/apps/products/forms.py

```diff
@@ -22,7 +22,7 @@ def clean(self, value):
 
 class ProductForm(forms.Form):
     name = forms.CharField(max_length=100)
-    code = forms.CharField(label=ugettext_noop("Product ID"), max_length=10, required=False)
+    code = forms.CharField(label=ugettext_noop("Product ID"), max_length=100, required=False)
```

Comment:
@gcapalbo Any idea what this was about?  My best guess is sms message length constraints or something.  I think the best course of action is this change, as you can create products with longer codes either through upload or just by leaving the field blank and having it autogenerated from the name.